pkg/binding/field_copy.go

```diff
@@ -0,0 +1,23 @@
+package binding
+
+import (
+	"bytes"
+	"strings"
+
+	"k8s.io/client-go/util/jsonpath"
+)
+
+func getValuesByJSONPath(obj map[string]interface{}, path_segments ...string) (string, bool, error) {
+	path := "{." + strings.Join(path_segments, ".") + "}"
```

Comment:
if we need path onward and do not use path_segements anywhere else, then let's pass to the function as argument path directly. This would also remove the need to split the path before calling this method.